ngx-fudis/projects/ngx-fudis/src/lib/components/expandable/readme.mdx

```diff
@@ -0,0 +1,62 @@
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import { linkTo } from '@storybook/addon-links';
+import { ExpandableComponent } from './expandable.component';
+import * as ExpandableStories from './expandable.stories.ts';
+
+<Meta title="Components/Expandable" />
+
+# Expandable
+
+Fudis expandable has two visual types: `variant: “regular”` and `variant: “lite”`, from which the latter is a lighter, link looking version of the component.
+
+The expandable is collapsed initially by default; this can be controlled with the collapsed input property.
+
+Expandable content and additional header buttons are provided via content projection by using `fudisContent` and `fudisAction` directives:
+
+```
+<fudis-expandable>
+  <ng-template fudisActions type="expandable">
+    <fudis-button/>
+  </ng-template>
+
+  <ng-template fudisContent type="expandable">
+    <your-body-template/>
+  </ng-template>
+</fudis-expandable>
+```
+
+All buttons wrapped in the fudisActions ng-template will appear to the right side of the header bar and on sm-sized screen they move to their own level.
+
+The content of the expandable is only initialized when the expandable is expanded for the first time. This helps to avoid unnecessary requests to the backend (among other expensive operations). Subsequent collapsing/expanding only hides the content instead of removing it from the DOM, for the same reason.
+
+About using lazy initalisation with content projection, see https://angular.io/guide/content-projection#conditional-content-projection for more background.
```

Comment:
Tän suoran linkin sijaan, suosittelen että annat tälle sisällölle kuvaavan nimen ja käytät tälläistä syntaksia
Esim. `[Angular example of conditional content projection](https://angular.io/guide/content-projection#conditional-content-projection)`